app/components/schools/participants/status_table_row.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Schools
+  module Participants
+    class StatusTableRow < BaseComponent
+      with_collection_parameter :profile
+
+      def initialize(profile:)
+        @profile = profile
+      end
+
+      def ineligible_participant?
+        profile.ecf_participant_eligibility&.ineligible_status?
```

Comment:
A bit pedant-y but could these return `true` or `false` - this could return `nil`